src/resource_clients/run.ts

```diff
@@ -138,6 +141,34 @@ export class RunClient extends ResourceClient {
         return cast(parseDateFields(pluckData(response.data)));
     }
 
+    /**
+     * https://docs.apify.com/api/v2#/reference/actor-runs/charge-run/charge-run
+     */
+    async charge(options: RunChargeOptions): Promise<ApifyResponse<Record<string, never>>> {
```

Comment:
Out of curiosity - what's the expected return type of this method? I see that most of the `RunClient` methods return `ActorRun` object (which reflects the changes made by the call). 

Is this return type (`ApifyResponse<Record<string, never>>`) a temporary thing before we figure out the actual API response shape, or is this the final thing?

Reply:
We discussed this in person. The endpoint uses a memorized version of the run from the database for performance reasons, as the run's pricing model cannot change during the run. Therefore we cannot return the modified run object and the only important output is the response status code.